### ISSUE TITLE
In Package file can a screenshot be added?

### ISSUE THREAD

**Issue by tclaus** [tclaus/stocks#40]
The package viewer  has space for a screenshot - how to get this into the package file?

**Comment by hfsfox** [comment]
No, Depot reads metadata from server, but it can placed to documentation folder for example
